.github/workflows/pythonpackage.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.9.x, 3.10.x, 3.11.x]
-        django-version: ['<4', '>=4']
+        python-version: [3.10.x, 3.11.x, 3.12.x]
+        django-version: ['<4', '>=4', '>=5']
```

Comment:
I checked the actions and it looks like `>=4` also resolves to Django 5, so I think we need to change that to something like `>4,<5` -- I tested that and it worked:

```bash
$ pip install "Django>4,<5"
...
Successfully installed Django-4.2.8
```